pages/about/research/recruitment/general-public.md

```diff
@@ -0,0 +1,77 @@
+---
+permalink: /about/research/recruitment/general-public/
+layout: styleguide
+title: Recruitment for general public
+category: About
+lead: Help shape the future of U.S. government websites.
+redirect_from:
+  # - /whats-new/research/
```

Comment:
Do we need redirects/aliases for this? For example, redirect from `research/recruitment/public/` to make it easier to type.

Reply:
Added redirect for `/research/recruitment/public` in 818cfd3.